.github/workflows/windows.yml

```diff
@@ -21,8 +21,6 @@ jobs:
         with:
           path: bridge/webui_bridge.h
           key: ${{ runner.os }}-${{ github.sha }}-bridge
-      - name: Install OpenSSL
-        run: choco install openssl -y
```

Comment:
Removing since openssl is installed by default on runner images (ref. https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md)

Also, having this in this separate job doesn't affect the build job. The build job runs without this installation already. The effect of having this is only adding a delay to the CI